### ISSUE TITLE
Orbit conversion

### ISSUE THREAD

**Issue by hannorein** [hannorein/rebound#72]
One thing that's a little confusing right now is that the `Orbit` class has a different convention for the attributes compared to the argument that the `Particle` class takes in the constructor. 

Orbit class has:
 - mean longitude
 - true anomaly

Whereas Particle class takes:
 - mean anomaly
 - true anomaly

I also think the true anomalies in those two classes might be defined differently (at least for circular orbits).

**Comment by hannorein** [comment]
I think this is part of the problem:
https://github.com/hannorein/rebound/blame/master/rebound/particle.py#L419

Not sure how to fix it. How does one define the longitude for a circular orbit? One should probably come up with an arbitrary reference axis. But that would also be somewhat unsatisfactory because what if the planet then gains a tiny bit of eccentricity, there would be a discontinuity.

**Comment by dtamayo** [comment]
Hi Hanno,

Yes, I also realized that. I think anither problem is the next line, which is incorrect. It should be the angle from the x axis to the particle's position, so I think that's a good angle at low e since it has no discontinuity. I may very well have introduced that error. I think the discontinuities are fine, as long as they preserve the physical angles, I.e. It's ok for omega and f to jump around for a circular orbit, as long as their sum gives the angle to ascending node.

I started going through the routine in detail again. I'm tired of going back to it and want to get it right this time, so I don't end up replicating the same work repeatedly. I'd like to also get it right to machine precision, since it matters a bit for the damping forces.  I can see jumps in a when I precess w (so everything else should be fixed) at the level of 10^-13, which is the accuracy of the orbital elements currently. Then I'll change python to call C, since the current setup is quite ugly. 

The edge cases are such a pain!

Dan
